meta-resin-common/recipes-support/resin-hostname/resin-hostname/resin-hostname

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ -z "$NEW_HOSTNAME" ]; then
+    NEW_HOSTNAME=$(jq -r '.uuid //empty' $CONFIG_PATH)
```

Comment:
This check is here, because resin-vars is a simple script that cannot depend on the uuid generation. Not only that but the uuid generation service needs resin-vars to get the CONFIG_PATH variable, so the only place to put this check was here. NEW_HOSTNAME should not be used to determine hostname.